src/browser/settings.jsx

```diff
@@ -1,7 +1,8 @@
 'use strict';
+const app = require('../package.json');
```

Comment:
@yuya-oc would your comments for use `require('../src/package.json')` apply here too?
 https://github.com/mattermost/desktop/pull/591/files#r137918302

Reply:
@dmeza No. #591 is a normal node.js script, so I suggested `require('../src/package.json')`. Please use Electron API here.